bde_env.drush.inc

```diff
@@ -82,7 +82,18 @@ function drush_bde_env_gen($filename = '') {
     ),
   );
 
-  $output = 'export BEHAT_PARAMS=' . escapeshellarg(json_encode($parameters, JSON_UNESCAPED_SLASHES));
+  switch (getenv('SHELL')) {
+    case '/usr/bin/fish':
```

Comment:
Are you sure this will work on all operating systems? Isn't the install location OS specific? This could just as well be `/bin/fish` or `/usr/local/bin/fish`.